### ISSUE TITLE
OS X build: incompatible pointer to integer conversion, library not found for -lcrt0.o

### ISSUE THREAD

**Issue by vdm** [micha/json-table#2]
```
$ sw_vers -productVersion
10.11.2
$ make
cc -c -O3 jt.c -o jt.o
jt.c:115:11: warning: incompatible pointer to integer conversion initializing 'ssize_t' (aka 'long') with an expression of type
      'void *' [-Wint-conversion]
  ssize_t bytes_r      = NULL;
          ^              ~~~~
1 warning generated.
cc -static jt.o -o jt
ld: library not found for -lcrt0.o
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [jt] Error 1
$
```

Fix:

```
$ git branch -v
* master 82abafe docs
$ git diff
diff --git a/Makefile b/Makefile
index 75c5c0e..ef77d92 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@

 VERSION = 1.1.0
 CFLAGS  = -O3
-LDFLAGS = -static
+LDFLAGS =
 PREFIX  = /usr/local
 BINDIR  = $(PREFIX)/bin
 MANDIR  = $(PREFIX)/man/man1
diff --git a/jt.c b/jt.c
index a0959fc..59bf772 100644
--- a/jt.c
+++ b/jt.c
@@ -112,7 +112,7 @@ char *read_stream(FILE *in) {
   size_t  ret_size     = 0;
   size_t  ret_newsize  = 0;
   size_t  ret_capacity = BUFSIZ * 2.5;
-  ssize_t bytes_r      = NULL;
+  ssize_t bytes_r      = 0;
   int     fd           = fileno(in);

   ret = jmalloc(ret_capacity);
```

**Comment by b-rucel** [comment]
@vdm Getting the same error with 2.0.0 tag on Mac, Thanks for the work around :+1:

**Comment by alandipert** [comment]
I think this is closed by #4, please reopen if not
